HW7(Tests)/__mocks__/Async-functions.js

```diff
@@ -0,0 +1,51 @@
+// Using a mock to prevent accessing third-party API
+function getUsers() { 
```

Comment:
1) you create mock asynchronous function as a synchronous one, it is wrong.
2) it is better to indicate in the name of the file that it is a mock, otherwise it is easy to get confused in spoils. eg Async-functions.mock.js 
3) if you want to mock for a third-party function, then you need to mock exactly for - fetch('https://jsonplaceholder.typicode.com/users')

Reply:
1. Fixed.
2. Fully agree with you. But as I understood, the names of the "Async-functions.js" in the working folder and in the "__mocks__" folder must be the same. If I will change the name of the mock then I will need to change the name of the original file and that`s not worth it.
P.S. I could be wrong
3. Fixed.